### ISSUE TITLE
Add resource locator of page to PageRemovedEvent

### ISSUE THREAD

**Issue by alexander-schranz** [sulu/sulu#6198]
# Problem description

Currently you don't have access to the url when removing a page.

# Proposed solution

The url is a important information which you example use when push sulu content to a external library and is maybe used there as identifier, so it would be great that the PageRemovedEvent contains that url.

https://github.com/sulu/sulu/blob/8ce644508cf3d0f70f9cfe489aceab271298a492/src/Sulu/Bundle/PageBundle/EventListener/DomainEventSubscriber.php#L269-L276

Should be updated to something like:

```php
 $this->domainEventCollector->collect( 
     new PageRemovedEvent( 
         $document->getUuid(), 
         $document->getWebspaceName(), 
         $document->getTitle(), 
         $document->getLocale(),
         [
             'url' => $document->getResourceLocator(),
         ]
     ) 
 ); 
```

Think the url can then just be added to the payload.

**Comment by alexander-schranz** [comment]
Reserved for [victor_vasiloi](https://twitter.com/alex_s_/status/1590425318602739712).

**Comment by vvasiloi** [comment]
> Reserved for [victor_vasiloi](https://twitter.com/alex_s_/status/1590425318602739712).

Yep, you can assign it to me.

**Comment by vvasiloi** [comment]
@alexander-schranz to what extent should I test it?
Also, the `getResourceLocator` method from your example doesn't exists, should I use `getResourceSegment`?

**Comment by alexander-schranz** [comment]
@vvasiloi you are right I think it is `getResourceSegment`, which contains that part.

A Unit test is fine enough or adjust if already a test for the `DomainEventSubscriber` exist that the `url` is now set.

**Comment by alexander-schranz** [comment]
Implemented by @vvasiloi in #6906 🎉  will be part of upcoming 2.6 release :)
